### PR TITLE
Add compose-go

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xmnlab** [comment]
@felipepaes, I am adding you here as maintainer, could you confirm that you are ok about that?
thanks

**Comment by xmnlab** [comment]
compose-go is a Python package that wraps the original docker compose project that is written in GoLang.

the current docker-compose [1] is really old (v1.29), because in the past it was written in Python, but since v2 it is a golang package.

The compose-go repository can be found at https://github.com/osl-package-recipes/compose-go


[1] https://github.com/conda-forge/docker-compose-feedstock/blob/main/recipe/meta.yaml, https://pypi.org/project/docker-compose/

**Comment by felipepaes** [comment]
> @felipepaes, I am adding you here as maintainer, could you confirm that you are ok about that? thanks

Sure! I confirm.
Thanks for adding me here as a maintainer @xmnlab

**Comment by xmnlab** [comment]
btw we haven't tested compose-go on macos yet .. as it is not working here, I just removed the support for that

**Comment by xmnlab** [comment]
@conda-forge/help-python this package is ready for review. 
thanks!

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/compose-go/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+build:
+  entry_points:
+    - docker-compose = compose:docker_compose
+  noarch: generic
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("go") }}
```
[the remark this answers is not in the document]
Comment (xmnlab) [reply recipes/compose-go/meta.yaml:21]: hey @BastianZim , thanks for the review.

probably this pr could be closed in favor of https://github.com/conda-forge/docker-compose-feedstock, 
it is now support version 2.

but about your question, maybe it is a lack of understand from my side about the noarch, I didn't know that it was about the build, I just thought it was about the run stage.

thanks for the review.

**Pull request closed without merging**